addons/hr_timesheet/views/project_task_sharing_views.xml

```diff
@@ -50,7 +50,10 @@
                                     <div t-attf-class="oe_kanban_card oe_kanban_global_click">
                                         <div class="row">
                                             <div class="col-6">
-                                                <field name="employee_id"/>
+                                                <img t-att-src="kanban_image('hr.employee', 'avatar_128', record.employee_id.raw_value)"
+                                                    t-att-title="record.employee_id.value"
+                                                    t-att-alt="record.employee_id.value"
+                                                    class="oe_kanban_avatar o_avatar rounded me-2"/>
                                                 <strong><span><t t-esc="record.employee_id.value"/></span></strong>
```

Comment:
It would be safer to avoid changing the xml structure.

I would prefer doing that: 
```suggestion
                                                <field name="employee_id"/>
                                                <strong invisible="1"><span><t t-esc="record.employee_id.value"/></span></strong>
```

Reply:
Moreover, I don't think the portal can access to those avatar images, so it is a bit useless to add them, IMO. 🤔